### ISSUE TITLE
Fails injecting first pod

### ISSUE THREAD

**Issue by Kav91** [newrelic/k8s-webhook#29]
With a fresh redis deployment, the first pod throws the below error

`{"level":"error","ts":"2019-08-15T04:01:20.561Z","caller":"server/webhook.go:164","msg":"error during mutation","err":"config map: 'redis-newrelic-integrations-config', not found","stacktrace":"github.com/newrelic/k8s-webhook/src/server.(*Webhook).ServeHTTP\n\t/app/src/server/webhook.go:164\nnet/http.(*timeoutHandler).ServeHTTP.func1\n\t/usr/local/go/src/net/http/server.go:3155"}`

When I scale up to 15 the other 14 are injected and reporting successfully.

Have replicated this multiple times.

**Comment by cristianciutea** [comment]
Hello @Kav91 thanks for bringing this to our attention, I'm trying to reproduce this issue, but I'm not able to do it with the examples from this repo. Could you please share more details about your environment and the deployment files, in order to reproduce this?
Thank you!

**Comment by Kav91** [comment]
@cristianciutea no problem.

So I've easily reproduced the issue in a fresh k8s environment multiple times now. (Ramon has seen this too).

Example deploy with kops:
```
export KOPS_CLUSTER_NAME=imesh.k8s.local
bucket_name=kops-test
export KOPS_STATE_STORE=s3://${bucket_name}
kops create cluster --node-count=2 --node-size=t2.medium --zones=us-east-1a --name=${KOPS_CLUSTER_NAME}
kops update cluster --name ${KOPS_CLUSTER_NAME} --yes
kops validate cluster
```
- Wait for cluster to be ready.
- Install k8s infra integration as per the docs.
- Perform the webhook steps as included in this repo.
- Deploy the included redis yaml as in this repo.

First will fail, and all subsequent ones will succeed.

**Comment by cristianciutea** [comment]
Hello @Kav91 could you try to extract the ConfigMap to a separate yaml file and try to apply that first, to be sure that it is created before pod creation? We suspect that the issue is caused by the fact that the ConfigMap is not yet created at that point.
We will search for a way to improve this.

**Comment by Kav91** [comment]
@cristianciutea Yes that is the only reason why this error would occur with that particular message, so aware of this situation. 
Just wanted to raise the issue because this now is a potential failure case for many other users who will be using any of the current examples being provided in this repository. 

Thanks, having this split out will create a further additional step which is not ideal.

**Comment by cristianciutea** [comment]
Hello @Kav91 this issue should be fixed by our latest release of the webhook #36
